.github/workflows/deploy-unleash-api-token.yaml

```diff
@@ -23,6 +22,5 @@ jobs:
       - name: deploy unleash api-token to prod
         uses: nais/deploy/actions/deploy@v2
         env:
-          APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
```

Comment:
Trenger man ikke `permission:id-token` her ?

Reply:
Jo, kanskje, men trenger vi disse filene? Kommer uansett ikke til å kjøre deploy på disse i prod. hvis vi legger inn dette nå. Har kun blitt kjørt en gang. 

Tror ikke vi kommer til å kjøre disse med det første - finnes egen favrooppgave for å vurdere om de har noen verdi og evt. slette filene.